src/main/java/br/com/caelum/comidinhas/tipoCozinha/TipoCozinhaController.java

```diff
@@ -0,0 +1,67 @@
+package br.com.caelum.comidinhas.tipoCozinha;
+
+import org.springframework.http.*;
+import org.springframework.web.bind.WebDataBinder;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import javax.validation.Valid;
+import java.net.URI;
+import java.util.List;
+
+import static java.lang.String.*;
+import static org.springframework.http.ResponseEntity.*;
+
+@RestController
+class TipoCozinhaController {
+
+    final TipoCozinhaRepository tipoCozinhaRepository;
```

Comment:
Atributo de dependência como private e final (já estava).

```suggestion
    private final TipoCozinhaRepository tipoCozinhaRepository;
```